lib/navigation/preprocess_imu.hpp

```diff
@@ -19,11 +22,32 @@ class ImuPreprocessor {
   SensorChecks checkReliable();
 
   template<std::size_t N>
-  core::Float getSpecificQuartile(const std::array<core::Float, N> &clean_accelerometer_data_copy,
-                                  core::Float quartile);
+  core::Float getSpecificQuartile(const std::array<core::Float, N> &clean_accelerometer_data,
```

Comment:
I don't like the naming of this function. Firstly, it doesn't return a `Quartile`. Secondly, it doesn't really give a quartile in the first place? If I call this function with `quartile == 0.14` this code still works (and may even be useful in some cases)! Find a better name that describes what this *actually* does.